### PR TITLE
Feat/creator catalyst dashboard

### PR DESCRIPTION
**Pull request by SupriyaShekar123**
# What does this pull request do?
In this pull request a form to upload cofounder profile video is made and the video is submitted to Cloudinary account.
Another form is made with link to personality test. 

Idealists_giphy

### CODE REVIEW

**Review by DannyS-stack — CHANGES_REQUESTED** [review]
Good work Supriya! Maybe it would be a good idea to go over it 1 more time and remove all the console.logs.

File: src/App.js [hunk 1]
```diff
@@ -728,6 +731,38 @@ class App extends Component {
...
                 }}
               />
+              <Route
+                exact
+                path="/cofounderProfileVideo"
+                render={(props) => {
+                  return (
+                    <CofounderProfileVideo  
+                      // {...props}
+                      // authState={this.state.auth}
```
Comment (dashakrolik) [inline src/App.js:741]: delete unused code

File: src/App.js [hunk 2]
```diff
@@ -728,6 +731,38 @@ class App extends Component {
...
+              />
+              <Route
+                exact
+                path="/cofounderPersonalityTest"
+                render={(props) => {
+                  return (
+                    <CofounderPersonalityTest 
+                      // {...props}
+                      // authState={this.state.auth}
+                      // login={this.requestLoginUser}
```
Comment (dashakrolik) [inline src/App.js:758]: delete unused code

File: src/components/Catalyst/ProfileUpdateForm.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+/** @jsx jsx */
+import React, { useState } from 'react'
+import { jsx, css } from '@emotion/core';
+import request from 'superagent'
+// import { baseUrl } from '../../../constants';
```
Comment (dashakrolik) [inline src/components/Catalyst/ProfileUpdateForm.js:5]: delete unused code

File: src/components/Cofounder/CofounderPersonalityTest.js [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+     const [errorCheck,seterrorCheck]= useState("false")
+     const [successMsg,setSuccessMsg]= useState(false)
+
+    const textRegEx =  new RegExp("([A-Z]{4}$)");
+  
+    const handelSubmit = async(e)=>{
+        e.preventDefault()
+        console.log("Testing")
+        console.log(e.target.value)
+        setSuccessMsg(true)
```
Comment (dashakrolik) [inline src/components/Cofounder/CofounderPersonalityTest.js:22]: delete console.logs

File: src/components/Cofounder/CofounderPersonalityTest.js [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+    const resData = await response.json(); 
+    console.log(resData)
+    
+  
+    // Return response data  
+    return resData;  
+    
+    }
+
+    const setValue =(e)=>{
```
Comment (dashakrolik) [inline src/components/Cofounder/CofounderPersonalityTest.js:41]: Spaces

File: src/components/Cofounder/CofounderPersonalityTest.js [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+        let format = /[!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?0-9a-z]+/;
+        setTestResult(e.target.value)
+      
+        if(e.target.value.match(format) ){
```
Comment (dashakrolik) [inline src/components/Cofounder/CofounderPersonalityTest.js:45]: Spaces

File: src/components/Cofounder/CofounderPersonalityTest.js [hunk 7]
```diff
@@ -0,0 +1,262 @@
...
+            <br></br>
+            <label>Upload your result here</label>
+            <input
+                type = "text"
+                type="text"
+                value={testResult}
+                // required 
+                // requiredError="This field is required." 
+                // validations={{matchRegexp:textRegEx}} 
+                // validationErrors={{matchRegexp:'The result should contain 4 Capital letters.'}}
```
Comment (dashakrolik) [inline src/components/Cofounder/CofounderPersonalityTest.js:86]: delete unused code

**Comment by SupriyaShekar123** [comment]
> Good work Supriya! Maybe it would be a good idea to go over it 1 more time and remove all the console.logs.

Thank you Danny. Yes I will remove all console.logs and push the code.

File: src/components/Cofounder/CofounderProfileVideo.js [hunk 8]
```diff
@@ -0,0 +1,302 @@
...
+
+import { Link } from "react-router-dom";
+
+
+
+export default function CofounderProfileVideo(props) {
+  const [ProfileVideoState, setProfileVideoState] = useState('');
+//   const [selectedVideo,setSelectedVideo] = useState('')
+  const [previewSource,setPreviewSource] = useState('')
+  const[successMsg,setSuccessMsg] = useState(false)
```
Comment (dashakrolik) [inline src/components/Cofounder/CofounderProfileVideo.js:16]: Spaces

File: src/components/Cofounder/CofounderProfileVideo.js [hunk 9]
```diff
@@ -0,0 +1,302 @@
...
+  const [loading, setLoading] = useState(false)
+
+
+  const handelVideoInputChange = (e) =>{
+      const video = e.target.files[0]
+      
+      previewVideo(video)
+    }
+
+     const previewVideo = (video)=>{
```
Comment (dashakrolik) [inline src/components/Cofounder/CofounderProfileVideo.js:27]: spaces....

File: src/components/Cofounder/CofounderProfileVideo.js [hunk 10]
```diff
@@ -0,0 +1,302 @@
...
+    const file = await res.json()
+    setPreviewSource(!previewSource)
+    console.log(file)
+     setLoading(false)
+     setSuccessMsg(true)
+    setVideos(file.secure_url)
+    console.log(file.secure_url)
+}
+    const cancelVideo = (e)=>{
+    e.preventDefault()
```
Comment (dashakrolik) [inline src/components/Cofounder/CofounderProfileVideo.js:58]: Indentation, please fix it everywhere

**Review by dashakrolik — CHANGES_REQUESTED** [review]

File: src/components/Cofounder/CofounderProfileVideo.js (hunk 10 above)
Comment (SupriyaShekar123) [reply src/components/Cofounder/CofounderProfileVideo.js:58]: Okay, Dasha I will make the changes.

**Review by dashakrolik — APPROVED** [review]

**Comment by dashakrolik** [comment]
> Good work Supriya! Maybe it would be a good idea to go over it 1 more time and remove all the console.logs.

It's been updated!:)

**Comment by DannyS-stack** [comment]
Looks good, I merged it

**Pull request merged**